README.md

```diff
@@ -38,14 +38,16 @@ For other versions, check the git tags of this repository.
 
 All parameters are required:
 
- * `-j1`: Replace with your jobs parameter to make.
+ * `-j1`: Replace with your jobs parameter to cmake.
```

Comment:
Does CMake infer the jobs correctly if this is omitted? Getting rid of the jobs parameter would be a nice improvement to this script.

Reply:
It just falls back to the default for that build system. For ninja that is the number of cores, but for make that's just 1.

Reply:
CMake never fails to disappoint me.

Reply:
Alright here's my suggestion. We can still remove the `-j` parameter to the build script, and next to your readme addition, we can also remind users of the existence of the `CMAKE_BUILD_PARALLEL_LEVEL` environment variable. So, if the user does not bother with these options, it will still succeed, however they can take advantage of the options to try to make it build faster.

Reply:
Updated 👍 

Unrelated question: I'm looking to add a build.bat in a future PR to bootstrap things from Windows - the missing link for me is I'm not sure how the `zig+llvm+lld+clang-x64_64-windows-gnu` bundles that the CI uses are generated, is that process documented somewhere? Is it something like build llvm(msvc crt) with msvc -> build zig2(msvc crt) with msvc -> build llvm(gnu crt) with zig2 -> build zig with llvm(gnu crt) ?

Reply:
I run the build script on linux, and then use https://github.com/ziglang/release-cutter/blob/master/update-ci-tarballs.zig to create the zip file